src/managers/input/InputManager.cpp

```diff
@@ -825,6 +825,8 @@ void CInputManager::onMouseWheel(IPointer::SAxisEvent e) {
 
             delta = 15.0 * discrete * factor;
         }
+
+        refocus();
```

Comment:
this should not be refocus, because on focus follows mouse 0 or 2 it shouldnt refocus

additionally, why only on source of 0?

Reply:
Ahh, okay. So refocus forces it despite mouse_follow 0 or 1. Didnt think about it.

Then I would need a little help, because I thought `mouseMoveUnified` would handle the focusing but I'm not making it through this code-snippet because the mouse in fact didnt move (and refocuse = false):

```c++
    if (MOUSECOORDSFLOORED == m_vLastCursorPosFloored && !refocus)
        return;
``` 

Im still learning about this code-base but I dont know another way

Reply:
And for the source of 0: I just wanted to make the minimum change to fix this bug and not introduce others by changing their behavior. For source = 1 it still registers a click  (at least on my surface) as well as 4 when side clicking with the wheel. But yea sure, It could be probably done on all sources

Reply:
`simulateMouseMovement`

Reply:
Done. Tested it with follow_mouse = 0 aswell.